lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,262 @@
+ * @returns {Promise<import('probot').Octokit.PullsListFilesResponseItem[]>}
+ */
+const getAllChangedFiles = async (context) => {
+  const changedFiles = [];
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse}
+   */
+  const pullRequest = context.payload.pull_request;
+  // 100 is used here since that is the maximum files per
+  // request that can be obtained from the API.
+  const maxFilesPerPage = 100;
+
+  let totalFilesFetched = 0;
+  let currentPage = 1;
+  do {
+    const fileListParams = context.repo({
+      pull_number: pullRequest.number,
+      per_page: maxFilesPerPage,
+      page: currentPage,
+    });
+
+    const fileResponse = await context.github.pulls.listFiles(fileListParams);
+    changedFiles.push(...fileResponse.data);
+    totalFilesFetched += fileResponse.data.length;
+    currentPage++;
+  } while (totalFilesFetched < pullRequest.changed_files);
+
+  return changedFiles;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const getRegistryReminder = (newJobFiles, changedFiles) => {
+  let registryReminder = ', ';
+
+  const hasNotAddedFileToRegistry = newJobFiles.some(
+    (file) => {
+      const jobs = getNewJobsFromFile(file);
+      const jobName = getJobNameFromFileName(file.filename);
+      return jobs.some((job) => !isRegistryUpdated(
+        `${jobName}.${job}`, changedFiles ));
+    }
+  );
+
+  if (hasNotAddedFileToRegistry) {
+    const fileText = newJobFiles.length > 1 ? 'files' : 'file';
+    const jobRegistryLink = 'job registry'.link(
+      'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
+    registryReminder +=
+      'please add the new job ' + fileText + ' to the '+ jobRegistryLink +' and ';
+  }
+
+  return registryReminder;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ */
+const getJobNameString = (newJobFiles) => {
+  let jobNameString = '';
+  const jobNameArray = newJobFiles.map((file) =>
+    getJobNameFromFileName(file.filename)
+  );
+  if (jobNameArray.length === 1) {
+    jobNameString = ' The name of the job is ' + jobNameArray[0] + '.';
+  } else {
+    jobNameString = ' The jobs are ' + jobNameArray.join(', ') + '.';
+  }
+
+  return jobNameString;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ * @param {string} prAuthor - Author of the Pull Request
+ */
+const getCommentBody = (newJobFiles, changedFiles, prAuthor) => {
+  const registryReminder = getRegistryReminder(newJobFiles, changedFiles);
+  const jobNameString = getJobNameString(newJobFiles);
+  const newLineFeed = '<br>';
+  const serverJobsForm = 'server jobs form'.link(
+    'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+  const wikiLinkText = (
+    'Please refer to this guide for details'.link(
```

Comment:
Can we just link to this guide instead of the complete line.

Reply:
Done.